server/svix-server/src/lib.rs

```diff
@@ -135,3 +136,27 @@ pub async fn run(cfg: Configuration, listener: Option<TcpListener>) {
     worker_loop.expect("Error initializing worker");
     expired_message_cleaner_loop.expect("Error initializing expired message cleaner")
 }
+
+mod docs {
+    use axum::{
+        response::{Html, IntoResponse},
+        routing::get,
+        Json, Router,
+    };
+
+    pub fn router() -> Router {
+        Router::new()
+            .route("/docs", get(get_docs))
+            .route("/openapi.json", get(get_openapi_json))
```

Comment:
```suggestion
            .route("/api/v1/openapi.json", get(get_openapi_json))
```
Just to be compatible with what we have in prod.